com.unity.cinemachine/Editor/Menus/CinemachineMenu.cs

```diff
@@ -305,9 +305,13 @@ static CinemachineBrain GetOrCreateBrain()
                 return CinemachineCore.Instance.GetActiveBrain(0);
 
             // Create a CinemachineBrain on the main camera
-            Camera cam = Camera.main;
+            var cam = Camera.main;
             if (cam == null)
+#if UNITY_2023_1_OR_NEWER
+                cam = Object.FindAnyObjectByType<Camera>(FindObjectsInactive.Exclude);
```

Comment:
The other option would be to call FindFirstObjectOfType. This is slower, but that's what the deprecated call `FindObjectOfType()` did. 

`FindAnyObjectByType` is faster than `FindFirstObjectOfType`, but if more cameras are in the scene, `FindAnyObjectByType` may not find the same on subsequent calls. 

I think we don't need consistency in this case, so I chose the faster method.

Reply:
I think the slower version is better in this case.  We do want consistency, or we may end up creating multiple Brains if Camera.main is never set.